index.html

```diff
@@ -1,57 +1,134 @@
 <!DOCTYPE html>
 <html lang="en">
 
-	<!-- a head for all the heady/conceptual/meta stuff -->
+
 	<head>
 
-		<title><!-- your title goes here --></title>
+		<title>Cameron Priestley</title>
+
+
+		<script language="javascript" type="text/javascript" src="../p5/p5.min.js"></script>
```

Comment:
One small thing ... you'll notice that this is changed from the original file. These two `script` tags should read:

```
<script language="javascript" type="text/javascript" src="p5/p5.min.js"></script>		
<script language="javascript" type="text/javascript" src="p5/addons/p5.dom.js"></script>
```

You must have copied this over from the example js project, which had to go up a folder level to load p5.js.